docs/ansible/install.rst

```diff
@@ -0,0 +1,96 @@
+.. highlight:: console
+
+.. _ansibleInstallation-ref:
+
+Ansible Installation 
+====================
+This document describes how to prepare for and install the software required to leverage Ansible for provisioning cloud resources. 
+
+Install Ansible
+---------------
+1. Start your virtual environment::
+
+   $ source ~/venv/bin/activate
+
+2. Install Ansible using ``pip``::
+ 
+   $ pip install ansible
+
+
+3. Verify the Ansible version is 2.0 or greater:: 
+
+   $ ansible --version
+
+
+
+Enable and configure SSH on your laptop
+---------------------------------------
+1. If you are on a Mac, then following the instructions here_ to enable SSH.
+
+.. _here: https://support.apple.com/kb/PH25252?viewlocale=en_US&locale=en_US
+
+2. Generate an RSA key pair:: 
+
+   $ ssh-keygen -t rsa
+
+   NOTE: Accept the default file location (~/.ssh/id_rsa) and use an empty pass phrase (just hit carriage return twice).
+
+3. Set permissions on the ~/.ssh directory::
+
+   $ chmod 755 ~/.ssh
+
+4. Create an authorized keys file::
+
+   $ touch ~/.ssh/authorized_keys && chmod 644 ~/.ssh/authorized_keys
+
+5. Install the ssh-copy-id command::
+
+   $ brew install ssh-copy-id 
```

Comment:
Why do we use `ssh-copy-id` over `cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys`?